{{cookiecutter.github_repository}}/provisioner/roles/common/tasks/main.yml

```diff
@@ -15,6 +15,9 @@
 #  when: vm == 1
 
 # Ansible requires aptitude and python2 for various tasks
+- name: Set hostname
+  action: shell hostnamectl set-hostname {{ domain_name }}
```

Comment:
reason being it would get changed every time if a single machine has multiple hosts